app/src/main/shared/Settings.ts

```diff
@@ -1,7 +1,8 @@
+import { settings as defaultSettings } from "./settings/default";
 const environment = require('environmentSettings');
 const appName = require('appName');
 
-export const settings: Settings = environment.settings;
+export const settings: Settings = Object.assign({}, defaultSettings, environment.settings);
```

Comment:
Now we have a default set of settings, and the other files override, instead of every file having to specify all settings. About time!